client/src/pages/Post/Post.component.jsx

```diff
@@ -69,12 +72,22 @@ const Post = () => {
     'dd MMM yyyy HH:mm, zzzz',
   )
 
+  const { _, data: answers } = useQuery(
+    [GET_ANSWERS_FOR_POST_QUERY_KEY, post?.id],
+    () => getAnswersForPost(post?.id),
+  )
+
   return isLoading ? (
     <Spinner centerHeight="200px" />
   ) : (
     <Flex direction="column" height="100%">
       <PageTitle
         title={`${post.title} - ${agencyShortName?.toUpperCase()} FAQ - AskGov`}
+        description={
+          answers && answers.length > 0
+            ? answers[0]?.body.replace(/<[^>]*>?/gm, '') // .replace() uses regex to remove html tags
```

Comment:
can you use a sanitizer library -_-

Reply:
Okay :') Actually JSX already escapes all the HTML tags so there isn't any security concern I think. But to be safe 👍🏻

Reply:
Hm for this, I could only find a package `html-to-text`, but am facing some issues with it due to the version of webpack (refer to https://github.com/html-to-text/node-html-to-text/issues/230). Will take a break from this first to look at the previous comment.

Reply:
Ah I misread, you're just converting HTML to text for the description. okay, deal with react-helmet-async first =)

Reply:
Replaced with `sanitize-html` package as suggested.